docs-ref-services/keyvault-secrets-spring-boot-starter-readme.md

```diff
@@ -0,0 +1,213 @@
+azure.keyvault.keyvault2.client-key=put-a-azure-client-key-here
+azure.keyvault.keyvault2.tenant-id=put-a-azure-tenant-id-here
+```
+Note if you decide to use multiple key vault support and you already have an
+existing configuration, please make sure you migrate that configuration to the
+multiple key vault variant. Mixing multiple key vaults with an existing single
+key vault configuration is a non supported scenario.
+
+### Case sensitive key mode
+To enable case sensitive mode, you can set the following property in the `appliation.properties`:
+```
+azure.keyvault.case-sensitive-keys=true
+```
+If your Spring property is using a name that does not honor the Key Vault secret key limitation use placeholders in properties. An example of using a placeholder:
+```
+my.not.compliant.property=${myCompliantKeyVaultSecret}
+```
+
+The application will take care of getting the value that is backed by the 
+`myCompliantKeyVaultSecret` key name and assign its value to the non compliant
+`my.not.compliant.property`.
+
+## Troubleshooting
+### Enable client logging
+Azure SDKs for Java offer a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
+
+### Enable Spring logging
+Spring allow all the supported logging systems to set logger levels set in the Spring Environment (for example, in application.properties) by using `logging.level.<logger-name>=<level>` where level is one of TRACE, DEBUG, INFO, WARN, ERROR, FATAL, or OFF. The root logger can be configured by using logging.level.root.
+
+The following example shows potential logging settings in `application.properties`:
+
+```properties
+logging.level.root=WARN
+logging.level.org.springframework.web=DEBUG
+logging.level.org.hibernate=ERROR
+```
+
+For more information about setting loging in pring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+ 
+
+## Next steps
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Key Vault Secrets](https://github.com/Azure/azure-sdk-for-java/tree/b2976bf8e5091b67593a98f0e6ff0c1478ab8935/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-secrets)
+
+## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](https://github.com/Azure/azure-sdk-for-java/tree/b2976bf8e5091b67593a98f0e6ff0c1478ab8935/sdk/spring/CONTRIBUTING.md) to build from source or contribute.
```

Comment:
Is there a reason to use a specific commit tag ? why not use master https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/CONTRIBUTING.md.

Reply:
Relative links likely got corrected. I'd check the contributing link in repo.